pkg/api/nextdate/nextdate.go

```diff
@@ -0,0 +1,140 @@
+package api
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const DF = "20060102"
+
+func getLastDayOfMonth(t time.Time) int {
+	return time.Date(t.Year(), t.Month()+1, 0, 0, 0, 0, 0, t.Location()).Day()
+}
+
+func NextDate(now time.Time, dstart, repeat string) (string, error) {
+	date, err := time.Parse(DF, dstart)
+	if err != nil {
+		return "", fmt.Errorf("Error dayStart: %v", err)
```

Comment:
1. При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
2. Нужно писать с маленькой буквы
```suggestion
		return "", fmt.Errorf("error dayStart: %w", err)
```

Reply:
Нужно везде поправить